python/whylogs/extras/image_metric.py

```diff
@@ -244,6 +249,24 @@ def zero(cls, config: Optional[MetricConfig] = None) -> "ImageMetric":
         )
 
 
+def init_image_schema(column_prefix: str = "image") -> DatasetSchema:
```

Comment:
```suggestion
def init_image_schema(column_prefix: str = "image") -> DatasetSchema:
  return DeclarativeSchema().add_resolver_spec(column_name=column_prefix, [MetricSpec(ImageMetric)])
```

Reply:
@richard-rogers tried to take your changes but the image metrics were missing in the dataset profile after that, which I was luckily testing for.